agent/php_execute.c

```diff
@@ -93,6 +93,12 @@
  * conditions wherever possible.
  */
 
+static inline nruserfn_t* nr_php_get_wraprec_from_op_array_extension(const char* fn, zend_function* zf) {
+  nruserfn_t* wraprec = (nruserfn_t*)ZEND_OP_ARRAY_EXTENSION(&zf->op_array, NR_PHP_PROCESS_GLOBALS(op_array_extension_handle));
```

Comment:
The cache slots are not available if the func is a trampoline so it should be checked before invoking `ZEND_OP_ARRAY_EXTENSION`
Note the safety instructions here:
https://github.com/php/php-src/blob/master/Zend/zend_extensions.c#L276-L289

Reply:
nr_php_get_wraprec_from_op_array_extension has been removed.